src/spox/opset/ai/onnx/v18.py

```diff
@@ -0,0 +1,3217 @@
+    class Attributes(BaseAttributes):
+        antialias: AttrInt64
+        axes: Optional[AttrInt64s]
+        coordinate_transformation_mode: AttrString
+        cubic_coeff_a: AttrFloat32
+        exclude_outside: AttrInt64
+        extrapolation_value: AttrFloat32
+        keep_aspect_ratio_policy: AttrString
+        mode: AttrString
+        nearest_mode: AttrString
+
+    @dataclass
+    class Inputs(BaseInputs):
+        X: Var
+        roi: Optional[Var]
+        scales: Optional[Var]
+        sizes: Optional[Var]
+
+    @dataclass
+    class Outputs(BaseOutputs):
+        Y: Var
+
+    op_type = OpType("Resize", "", 18)
+
+    attrs: Attributes
+    inputs: Inputs
+    outputs: Outputs
+
+
+class _ScatterElements(StandardNode):
+    @dataclass
+    class Attributes(BaseAttributes):
+        axis: AttrInt64
+        reduction: AttrString
+
+    @dataclass
+    class Inputs(BaseInputs):
+        data: Var
+        indices: Var
+        updates: Var
+
+    @dataclass
+    class Outputs(BaseOutputs):
+        output: Var
+
+    op_type = OpType("ScatterElements", "", 18)
+
+    attrs: Attributes
+    inputs: Inputs
+    outputs: Outputs
+
+
+class _ScatterND(StandardNode):
+    @dataclass
+    class Attributes(BaseAttributes):
+        reduction: AttrString
+
+    @dataclass
+    class Inputs(BaseInputs):
+        data: Var
+        indices: Var
+        updates: Var
+
+    @dataclass
+    class Outputs(BaseOutputs):
+        output: Var
+
+    op_type = OpType("ScatterND", "", 18)
+
+    attrs: Attributes
+    inputs: Inputs
+    outputs: Outputs
+
+
+class _Split(StandardNode):
+    @dataclass
+    class Attributes(BaseAttributes):
+        axis: AttrInt64
+        num_outputs: Optional[AttrInt64]
+
+    @dataclass
+    class Inputs(BaseInputs):
+        input: Var
+        split: Optional[Var]
+
+    @dataclass
+    class Outputs(BaseOutputs):
+        outputs: Sequence[Var]
+
+    op_type = OpType("Split", "", 18)
+
+    attrs: Attributes
+    inputs: Inputs
+    outputs: Outputs
+
+
+def bitwise_and(
+    A: Var,
+    B: Var,
+) -> Var:
+    r"""
+    Returns the tensor resulting from performing the bitwise ``and``
+    operation elementwise on the input tensors ``A`` and ``B`` (with
+    Numpy-style broadcasting support).
+
+    This operator supports **multidirectional (i.e., Numpy-style)
+    broadcasting**; for more details please check `the
+    doc <https://github.com/onnx/onnx/blob/main/docs/Broadcasting.md>`__.
+
+    Parameters
+    ==========
+    A
+        Type T.
+        First input operand for the bitwise operator.
+    B
+        Type T.
+        Second input operand for the bitwise operator.
+
+    Returns
+    =======
+    C : Var
+        Type T.
+        Result tensor.
+
+    Notes
+    =====
+    Signature: ``ai.onnx@18::BitwiseAnd``.
+
+    Type constraints:
+     - T: `tensor(int16)`, `tensor(int32)`, `tensor(int64)`, `tensor(int8)`, `tensor(uint16)`, `tensor(uint32)`, `tensor(uint64)`, `tensor(uint8)`
```

Comment:
These should be double ` `` ` in order to not offend sphinx. This will also update the existing modules, obviously. Maybe we better fix that in its own PR.

Reply:
Yes, I guess they could be. It doesn't necessarily look that bad, though. Left to follow-up.